src/java/htsjdk/samtools/example/PrintReadsExample.java

```diff
@@ -97,7 +97,7 @@ private static void printConfigurationInfo() throws IOException {
                 ' ' + (DeflaterFactory.usingIntelDeflater() ? "IntelDeflater" : "JdkDeflater"));
         log.info("CREATE_INDEX:" + Defaults.CREATE_INDEX +
                 ' ' + "CREATE_MD5:" + Defaults.CREATE_MD5 +
-                ' ' + "USE_ASYNC_IO:" + Defaults.USE_ASYNC_IO +
+                ' ' + "USE_ASYNC_IO_FOR_SAMTOOLS:" + Defaults.USE_ASYNC_IO_FOR_SAMTOOLS +
                 ' ' + "BUFFER_SIZE:" + Defaults.BUFFER_SIZE +
                 ' ' + "COMPRESSION_LEVEL:" + Defaults.COMPRESSION_LEVEL +
                 ' ' + "NON_ZERO_BUFFER_SIZE:" + Defaults.NON_ZERO_BUFFER_SIZE +
```

Comment:
done, added a `Map<String, Object> allDefaults` to Defaults from which clients can easily make a toString. I felt that making a simple toString is maybe too constraining